phpunit.xml.dist

```diff
@@ -1,15 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit backupGlobals="false"
-         backupStaticAttributes="false"
-         bootstrap="./vendor/autoload.php"
-         colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnFailure="false"
-         syntaxCheck="false"
+<phpunit
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
```

Comment:
Yeah I personally prefer this style 👍 

However I'm not sure that referencing the XSD from vendor directory is a good idea: we use several versions of PHPUnit which might have different XSD files. I would rather use the online version, e.g. https://schema.phpunit.de/5.4/phpunit.xsd. WDYT?